extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -406,39 +417,37 @@ protected function sendEmails( array $watchers, WallNotificationEntity $notifica
 				wfRunHooks( 'NotificationGetMailNotificationMessage', [
 					&$notification, &$data, $key, $watcherName, $author_signature, $textNoHtml, $text
 				]);
-				if ( empty( $data ) ) {
-					$data = [
-						'$WATCHER' => $watcherName,
-						'$WIKI' => $notification->data->wikiname,
-						'$PARENT_AUTHOR_NAME' => 	(empty($notification->data->parent_displayname)
-													? ''
-													: $notification->data->parent_displayname),
-						'$AUTHOR_NAME' => $notification->data->msg_author_displayname,
-						'$AUTHOR' => $notification->data->msg_author_username,
-						'$AUTHOR_SIGNATURE' => $author_signature,
-						'$MAIL_SUBJECT' => wfMessage('mail-notification-subject', [
-							'$1' => $notification->data->thread_title,
-							'$2' => $notification->data->wikiname
-						])->text(),
-						'$METATITLE' => $notification->data->thread_title,
-						'$MESSAGE_LINK' =>  $notification->data->url,
-						'$MESSAGE_NO_HTML' =>  $textNoHtml,
-						'$MESSAGE_HTML' =>  $text,
-						'$MSG_KEY_SUBJECT' => $key,
-						'$MSG_KEY_BODY' => 'mail-notification-body',
-						'$MSG_KEY_GREETING' => 'mail-notification-html-greeting',
-					];
-				}
 
-				if ( !( $watcher->getBoolOption('unsubscribed') === true ) ) {
-					$this->sendEmail( $watcher, $data );
-				}
+				$this->sendEmail( $watcher, $data );
```

Comment:
Maybe we should put some kibana logging here to see if this code path is used after the release?  I don't think it should be and it would be nice to confirm.  Then we can take this out...